data.sql

```diff
@@ -0,0 +1,4 @@
+INSERT INTO animals(name, date_of_birth, weight_kg, neutered, escape_attempts) VALUES ('Agumon', '2020-02-03', 0, TRUE, 10.23);
+INSERT INTO animals(name, date_of_birth, weight_kg, neutered, escape_attempts) VALUES ('Gabumon', '2018-11-15', 8, TRUE, 2);
```

Comment:
- [ ] As requested kindly swap the data so its in the correct format for users.

Your Code
```

INSERT INTO animals(name, date_of_birth, weight_kg, neutered, escape_attempts) VALUES ('Agumon', '2020-02-03', 0, TRUE, 10.23);
```

Expected Code

```

INSERT INTO animals(name, date_of_birth, weight_kg, neutered, escape_attempts) VALUES ('Agumon', '2020-02-03', 10.23, TRUE, 0);
```